src/vtools-ffprobe.py

```diff
@@ -266,10 +266,11 @@ def parse_ffprobe_output(out, debug):
         frame["bpp"] = (int(frame["pkt_size"]) * 8) / (
             int(frame["width"]) * int(frame["height"])
         )
-        # get the duration of this frame
-        # some file produce duration_time instead of pkt_duration_time
+        # pkt_duration_time is deprecated, use duration_time
```

Comment:
please move the code that knows how to get a timestamp from the ffprobe output (including which column to use, and which unit is each column) into a separate function, and use in both places

Reply:
I am not sure if I understand your suggestion. Reading ffprobe output only happens here.